playbooks/roles/ec2-snapshot/meta/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+dependencies: []
```

Comment:
@sidorovdmitry 
I suggest using meta/main.yml in all our roles (some kind a template...) it will become a good habit